react-app/src/components/TutorialOverview.tsx

```diff
@@ -0,0 +1,78 @@
+/********************************************************************************
+ * Copyright (c) 2020-2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ ********************************************************************************/
+import React, {useState} from 'react';
+import {makeStyles, Theme, createStyles} from '@material-ui/core/styles';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import StepperComponent from './StepperComponent';
+import {Grid, IconButton, StepContent} from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import {Tutorial} from '../../../schema/tutorial';
+import {vsTheme} from '../VsTheme';
+
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            width: '100%',
+        },
+        text: {
+            color: vsTheme.text.color,
+            textAlign: "left",
+        },
+    }),
+);
+
+interface TutorialOverviewProps {
+    tutorial: Tutorial;
+}
+
+const TutorialOverview = (props: TutorialOverviewProps) => {
```

Comment:
you can deconstruct in line:
```suggestion
const TutorialOverview = ({tutorial}:TutorialOverviewProps) => {
```